spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -40,8 +40,10 @@ the MicroProfile services the token will be accessed with.
 The required minimum set of MP-JWT headers and claims is:
 
 typ:: This JOSE header parameter identifies the token as an RFC7519 and must be "JWT" https://tools.ietf.org/html/rfc7519#section-5.1[RFC7519, Section 5.1]
-alg:: This JOSE header parameter identifies the cryptographic algorithm used to secure the JWT. RSASSA-PKCS1-v1_5 SHA-256 algorithm is required for signing the claims and must be specified as "RS256", https://tools.ietf.org/html/rfc7515#section-4.1.1[RFC7515, Section 4.1.1]. RSAES using Optimal Asymmetric Encryption Padding and AES in Galois/Counter Mode (GCM) algorithms are required for
-encrypting the claims and must be specified as "RSA-OAEP", https://tools.ietf.org/html/rfc7518#section-4.3[RFC7518, Section 4.3] and "A256GCM", https://tools.ietf.org/html/rfc7518#section-5.3[RFC7518, Section 5.3]
+alg:: This JOSE header parameter identifies the cryptographic algorithm used to secure the JWT.
```

Comment:
Adding this is a breaking change, ES256 support only in 2.0?

Reply:
> Adding this is a breaking change, ES256 support only in 2.0?
It is not related to `ES256`. `alg` is already specified in MP JWT 1.1.1, I've just updated this text to clarify what this property contains when the token is encrypted.

Reply:
I misread the diff, the addition doesn't seems to be a problem.